packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -31,31 +33,39 @@ abstract contract FundingRateApplier is Testable, Lockable {
      * FUNDING RATE APPLIER DATA STRUCTURES *
      ****************************************/
 
-    // Locally stores the finder.
-    // This is private because it is expected that another contract, like the FeePayer, will publicly expose the finder
-    // address.
-    FinderInterface private finder;
-
-    // Last time the `cumulativeFundingRateMultiplier` was updated.
-    uint256 public lastUpdateTime;
+    struct FundingRate {
+        // Current funding rate value.
+        FixedPoint.Signed rate;
+        // Identifier to retrieve the funding rate.
+        bytes32 identifier;
```

Comment:
Would this `identifier` ever != `fundingRateIdentifier`?

Reply:
Nope, just moved the funding rate identifier to this struct.

Reply:
Can we move the fundingRateIdentifier state var to this contract? I know right now its in the FeePayer and removing it from there would break tests, but once we deprecate the FRS can we add it here?

Reply:
Sorry, I'm a little confused. The other one exists in the fundingRateFeePayer, but that entire contract will be deprecated after this PR as it's no longer in the inheritance tree for the perp.

So I was thinking it makes sense to just leave that contract be and remove it along with the FRS in a later PR. Thoughts?

Reply:
Agreed we will remove `FundingRatePayer` contract, so in that process are you saying we'll move the `fundingRateIdentifier` into this contract?

Reply:
Or do you want to just pass it here by construction, looks like no other contract in the Perp system needs access to the `fundingRateIdentifier` so it might be cleaner to just store the identifier at the state level. WDYT